node/src/test_utils/neighborhood_test_utils.rs

```diff
@@ -92,7 +93,7 @@ pub fn neighborhood_from_nodes(
                 root.node_addr_opt().unwrap(),
                 vec![NodeDescriptor::from((
                     neighbor,
-                    DEFAULT_CHAIN_ID == chain_id_from_name(DEFAULT_CHAIN_NAME),
+                    Blockchain::EthRopsten, //TODO this is weird - it always was like wrong TEST_DEFAULT_CHAIN_ID and default_chain_name didn't point to the same thing (maybe it was intended but bad names used)
```

Comment:
I don't believe this was always wrong. I can't recall exactly but I suspect these tests were to prove that the production values `"mainnet"`, network ID 1 along with the mainnet Contract Address would work properly when used with CryptDE mechanisms for signing an verifying signatures for network gossip.